client/src/browser_action/browser_action.html

```diff
@@ -1,17 +1,23 @@
-<!doctype html>
-<style type="text/css">
-	#mainPopup {
-		padding: 10px;
-		height: 200px;
-		width: 400px;
-		font-family: Helvetica, Ubuntu, Arial, sans-serif;
-	}
-	h1 {
-		font-size: 2em;
-	}
-</style>
+<!DOCTYPE html>
+<html>
 
-<div id="mainPopup">
-<h1>Hello extensionizr!</h1>
-<p>To shut this popup down, edit the manifest file and remove the "default popup" key. To edit it, just edit ext/browser_action/browser_action.html. The CSS is there, too.</p>
-</div>
\ No newline at end of file
+<head>
+	<style type="text/css">
+		body {
+			padding: 10px;
+			width: 500px;
+			height: 500px;
+		}
+	</style>
+
+	<script async src="popup.js">
```

Comment:
It was having troubles loading otherwise. I am not sure if I have resolved it if we have removed it.